docs/docs/examples/logging/efk/EFK.md

```diff
@@ -206,27 +209,38 @@ $ cat <<EOF> ~/environment/logging/elasticsearch_domain.json
     }
 }
 EOF
+```
+Initialize the Elasticsearch domain using the `elasticsearch_domain.json`
 
-$ aws es create-elasticsearch-domain \
-   --cli-input-json   file://~/environment/logging/es_domain.json
+```shell
+aws es create-elasticsearch-domain \
+   --cli-input-json   file://~/environment/logging/elasticsearch_domain.json
 ```
 
-It takes a while for Elasticsearch clusters to change to an active state. Check the AWS Console to see the status of the cluster, and continue to the next step when the cluster is ready.
+!!!info ""
+    
+    It takes a while for Elasticsearch clusters to change to an active state. Check the AWS Console to see the status of the cluster, and continue to the next step when the cluster is ready.
 
-*Configure Elasticsearch access:* At this point you need to map roles to users in order to set fine-grained access control, because without this mapping all the requests to the cluster will result in permission errors. You should add the Fluent Bit ARN as a backend role to the `all-access` role, which uses the Elasticsearch APIs. To find the Fluent Bit ARN run the following command and export the value of `ARN Role` into the `FLUENTBIT_ROLE` environment variable:
+At this point you need to map roles to users in order to set fine-grained access control, because without this mapping all the requests to the cluster will result in permission errors. You should add the `Fluentbit` ARN as a backend role to the `all-access` role, which uses the Elasticsearch APIs. To find the `fluentbit` ARN run the following command and export the value of `ARN Role` into the `FLUENTBIT_ROLE` environment variable:
+```shell
+eksctl get iamserviceaccount --cluster dcd-ap-southeast-2
+```
+The output of this command should look similar to this:
 ```shell
-$ eksctl get iamserviceaccount --cluster dcd-ap-southeast-2
-[ℹ] eksctl version 0.37.0
-[ℹ] using region ap-southeast-2
 NAMESPACE    NAME                ROLE ARN
 kube-system cluster-autoscaler   arn:aws:iam::887464544476:role/eksctl-dcd-ap-southeast-2-addon-iamserviceac-Role1-1RSRFV0BQVE3E
```

Comment:
I think we should redact or change the dcd account number with a random number as this documents are public (here and anywhere else we used it in samples)